db/migrate/20150402153201_create_votes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateVotes < ActiveRecord::Migration
+  def change
+    create_table :votes do |t|
+      t.boolean :upvote
```

Comment:
This column was removed and the table changed in a later migration, no longer using a boolean